Client/Assets/MainFrameAtDocumentEnd.js.LICENSE.txt

```diff
@@ -0,0 +1 @@
+/*! https://mths.be/punycode v1.4.1 by @mathias */
```

Comment:
Could we remove this file as it is not required for this issue.
Everything else looks good 👍